examples/vue/src/pages/ui/components/authenticator/custom-sign-up-fields/index.vue

```diff
@@ -31,7 +31,9 @@ const services = {
   <authenticator initial-state="signUp" :services="services">
     <template v-slot:sign-up-fields>
       <authenticator-sign-up-form-fields />
-      <amplify-check-box :errorMessage="validationErrors.acknowledgement" />
+      <amplify-check-box
+        :errorMessage="validationErrors.acknowledgement as string"
```

Comment:
Since `validationErrors.acknowledgement` is a bit of an unusual type `string or string[]` we have to typecast this as a string. Not sure what sort of version bump we should do for this. @reesscot  @calebpollman @wlee221

Reply:
This does match Reacts implementation, it also has to cast

Reply:
I believe this is a technically a breaking change, because existing customer codes that don't cast to `string` will be broken.

We could see it as a bug, but since this was a code snippet that is in docs [here](https://ui.docs.amplify.aws/vue/connected-components/authenticator/customization#sign-up-fields), we should be cautious about this.